lib/modules/neverEndingReddit.js

```diff
@@ -110,31 +113,26 @@ addModule('neverEndingReddit', {
 
 			this.allLinks = RESUtils.$things();
 
+			var nextLink = nextPrevLinks.next;
+			if (nextLink) {
+				this.nextPageURL = nextLink.getAttribute('href');
+				var nextXY = RESUtils.getXYpos(nextLink);
+				this.nextPageScrollY = nextXY.y;
 
-			// get the first link to the next page of reddit...
-			var nextPrevLinks = modules['neverEndingReddit'].getNextPrevLinks(this.siteTable);
```

Comment:
Removing `this.siteTable` from this call won't work because search pages have multiple "next" arrows, ugh.